### DISCUSSION TITLE
Dot/Bar mode for CCLEDBar

### DISCUSSION THREAD

**Discussion by ovinas** [tttapa/Control-Surface#522] [Q&A]
Absolute fantastic project and I love playing around with it. Not being a programmer it's sometimes a bit tricky for me but trial & error works fine in most cases.
But I can't find out how to set the Dot mode for CCLEDBar. Any help appreciated. Thanks.

This works fine in Bar mode.
CCLEDBar<9> leds { {{5, 6, 7, 8, 9, 10, 11, 12, 13}}, {5, CHANNEL_1}};

**Comment by tttapa** [comment]
You should be able to add `leds.dotMode();` to your setup.

**Reply by ovinas** [reply]
Wow, that was a quick answer. And it works fine. Thanks a lot.
Any link for a donation to the project?

**Reply by EvgeniySavkin** [reply]
Peter needs to think about a blog where he would communicate in his free time(which he does not have))) with users and register a donation. I already have so many orders for midi controllers. But I am afraid to make them for money, since it is unethical in relation to the author of the library. Peter, I ask you and I create a platform for depositing funds to your account.

**Reply by tttapa** [reply]
I've been thinking about this for a long time, but I'm unsure that I'll have enough free time to provide the support that might be expected by people who donate, it's just a hobby project after all.

I noticed that GitHub Sponsors now allows for one-time donations, I'll look into it again.
